doc/classes/EditorSceneFormatImporter.xml

```diff
@@ -37,31 +37,35 @@
 			<return type="Animation" />
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="flags" type="int" />
-			<argument index="2" name="bake_fps" type="int" />
+			<argument index="2" name="options" type="Dictionary" />
+			<argument index="3" name="bake_fps" type="int" />
 			<description>
 			</description>
 		</method>
 		<method name="_import_scene" qualifiers="virtual">
 			<return type="Object" />
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="flags" type="int" />
-			<argument index="2" name="bake_fps" type="int" />
+			<argument index="2" name="options" type="Dictionary" />
+			<argument index="3" name="bake_fps" type="int" />
 			<description>
 			</description>
 		</method>
-		<method name="import_animation_from_other_importer">
+		<method name="do_import_animation">
 			<return type="Animation" />
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="flags" type="int" />
-			<argument index="2" name="bake_fps" type="int" />
+			<argument index="2" name="options" type="Dictionary" />
+			<argument index="3" name="bake_fps" type="int" />
 			<description>
 			</description>
 		</method>
-		<method name="import_scene_from_other_importer">
+		<method name="do_import_scene">
```

Comment:
This method should be removed because its no longer needed.

Reply:
Done. Note that as a consequence of removing this, it would not be possible to manually invoke the FBX or COLLADA importers from GDScript.

For me, however, since I only ever use glTF, and GLTFDocument has its own `import_scene` (which takes a GLTFState), I don't need this general API.

Given that glTF is the best supported document format, it stands to reason that having GLTFDocument exposed will satisfy most use cases.